base/some.jl

```diff
@@ -138,8 +138,7 @@ true
 macro something(args...)
     expr = :(nothing)
     for arg in reverse(args)
-        expr = :((val = $arg) !== nothing ? val : $expr)
+        expr = :(val = $arg; $(GlobalRef(Base, :isnothing))(val) ? $expr : val)
     end
     return esc(:(something(let val; $expr; end)))
```

Comment:
If we're using `Base.isnothing` we may as well be consistent
```suggestion
    return esc(:(Base.something(let val; $expr; end)))
```

Reply:
I prefer using `GlobalRef`, but yeah, that should be better.

Reply:
There seemed to be many missing GlobalRef though (and you would still need one on `Base` there), so esc is likely easier